tracer/test/Datadog.Trace.Tests/CallTarget/ValueTaskAsyncContinuationGeneratorTests.cs

```diff
@@ -0,0 +1,345 @@
+// <copyright file="ValueTaskAsyncContinuationGeneratorTests.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#if NETCOREAPP3_1_OR_GREATER
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Datadog.Trace.ClrProfiler.CallTarget;
+using Datadog.Trace.ClrProfiler.CallTarget.Handlers.Continuations;
+using Xunit;
+
+namespace Datadog.Trace.Tests.CallTarget
+{
+    public class ValueTaskAsyncContinuationGeneratorTests
```

Comment:
Approving this file since this is identical to the updated `ValueTaskContinuationGeneratorTests`, after find/replace `ValueTaskContinuationGeneratorTests` with `ValueTaskAsyncContinuationGeneratorTests` and changing the OnAsyncMethodEnd's to return `Task<TReturn>` instead of `TReturn`